src/containers/Base.js

```diff
@@ -20,36 +20,41 @@ export default class Base extends React.PureComponent {
   }
 
   componentDidMount () {
-    this.intitialize()
+    this.initialise()
```

Comment:
`color` we don't have a choice, `initialise` we do, and because we're 🇦🇺 , I think we can be a little proud and roll with it (it's a private method after all).